packages/@react-spectrum/button/docs/Button.mdx

```diff
@@ -84,6 +84,31 @@ function Example() {
 }
 ```
 
+## Pending
```

Comment:
This may be follow-up, but will other buttons support the pending state? (i.e. action button)